src/components/v5/frame/ColonyHome/partials/FundsCards/hooks.ts

```diff
@@ -142,3 +143,19 @@ export const useTotalFunds = () => {
 
   return totalFunds;
 };
+
+export const useIsAddNewTeamVisible = () => {
+  const subTeams = useSubDomains();
+
+  const selectedDomain = useGetSelectedDomainFilter();
+
+  // In case "All teams" selected we have "General" team created by default
+  // @TODO: probably with nested teams this functionality will change
+  if (!selectedDomain && (!subTeams || subTeams.length <= 1)) {
+    return true;
+  }
+  const isAddNewTeamVisible =
+    selectedDomain?.isRoot && (!subTeams || subTeams.length < 1);
+
+  return isAddNewTeamVisible;
```

Comment:
Would it make sense to directly return `selectedDomain?.isRoot && (!subTeams || subTeams.length < 1)` without creating the `isAddNewTeamVisible`?

Reply:
good spot, thank you!